### DISCUSSION TITLE
Missing docs for `9.4.3's` new `navigation.instant.progress` feature?

### DISCUSSION THREAD

**Discussion by HonkingGoose** [squidfunk/mkdocs-material#6119] [General]
The [Material for Mkdocs manual for setting up navigation](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/) does not mention the new `navigation.instant.progress` feature. See [changelog for `mkdocs-material-9.4.3`](https://github.com/squidfunk/mkdocs-material/releases/tag/9.4.3). The docs mention the other new `navigation.instant.prefetch` feature though.

I'll let you decide what to do. Maybe you meant for `navigation.instant.progress` to stay out of the docs? 🙃

**Comment by squidfunk** [comment]
Wow, the project has reached a size, that there are so many eyes on it that everything that I do seems to get reviewed by hundreds of eyes the moment I push some code/release 😅 We've already added it 10m after the release in ddd125ee744fe25fb0a78c809406d9fa9fb57ae2 and it's [already online](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#progress-indicator).

**Reply by HonkingGoose** [reply]
Thank you for the quick docs update! ❤️
